docs/deploying.adoc

```diff
@@ -40,7 +40,7 @@ metadata:
   name: standalone-etl
 spec:
   image: quay.io/streamshub/flink-sql-runner:0.1.0
-  flinkVersion: v1_20
+  flinkVersion: v2_0
```

Comment:
This won't work until the image ref points to one that has the flink 2.0 binaries in.